Makefile.am

```diff
@@ -38,6 +38,8 @@ AM_CPPFLAGS = \
 	-include $(top_builddir)/config.h \
 	-I$(top_builddir) \
 	-I$(top_srcdir) \
+	-I$(top_srcdir)/src/mavlink-router \
+	-I$(top_srcdir)/src/common \
```

Comment:
That's a long standing issue that I'd like to be fixed for good. I prefer fixing the issue first.